assets/js/components/legacy-notifications/DashboardCoreSiteAlerts.stories.js

```diff
@@ -0,0 +1,118 @@
+/**
+ * DashboardCoreSiteAlerts Component Stories.
+ *
+ * Site Kit by Google, Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * Internal dependencies
+ */
+import WithRegistrySetup from '../../../../tests/js/WithRegistrySetup';
+import DashboardCoreSiteAlerts from './DashboardCoreSiteAlerts';
+import { CORE_SITE } from '../../googlesitekit/datastore/site/constants';
+import { CORE_USER } from '../../googlesitekit/datastore/user/constants';
+
+const Template = ( { setupRegistry } ) => (
+	<WithRegistrySetup func={ setupRegistry }>
+		<DashboardCoreSiteAlerts />
+	</WithRegistrySetup>
+);
+
+const notification1 = {
+	id: 'test-notification',
+	title: 'test-title',
+	content: 'test-content',
+	ctaURL: 'test-cta-url',
+	ctaLabel: 'test-cta-label',
+	ctaTarget: 'test-cta-target',
+	learnMoreURL: 'test-learn-more-url',
+	learnMoreLabel: 'test-learn-more-label',
+	dismissible: true,
+	dismissLabel: 'test-dismiss-label',
```

Comment:
Can we use slightly more realistic data for these stories? These end up looking a bit odd and are less instructive/useful as references. How about:

```suggestion
	id: 'test-notification',
	title: 'Google Analytics 5 Beta',
	content: 'Upgrade to the latest and greatest version of Analytics!',
	ctaURL: '#ga5-upgrade',
	ctaLabel: 'Upgrade to GA5!',
	ctaTarget: '_blank',
	learnMoreURL: '#learn-more',
	learnMoreLabel: 'Learn more',
	dismissible: true,
	dismissLabel: 'Dismiss this message',
```

Reply:
Sure thing, I just grabbed the one we used in integration tests. https://github.com/google/site-kit-wp/blob/develop/tests/phpunit/integration/Core/Notifications/NotificationTest.php#L32-L45